src/internal/util/isReadableStreamLike.ts

```diff
@@ -0,0 +1,24 @@
+import { isFunction } from './isFunction';
+
+export interface ReadableStreamLike<T> {
+  getReader(): ReadableStreamDefaultReader<T>;
```

Comment:
I'll give this a proper review later, but I don't think we can rely on - or explicitly reference using a `///` comment - this type: `ReadableStreamDefaultReader<T>`

Reply:
Ah great catch. I'll stub it out too since it's easy.

Reply:
Done.